modules/integration_azure-app-service/conf/02-http-5xx-errors-count.yaml

```diff
@@ -0,0 +1,26 @@
+---
+module: "Azure App Service"
+name: "http 5xx error rate"
+filtering: "filter('resource_type', 'Microsoft.Web/sites') and filter('is_Azure_Function', 'false') and filter('primary_aggregation_type', 'true')"
+aggregation: ".sum(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+value_unit: "%"
+signals:
+  http5xx:
+    metric: Http5xx
+    extrapolation: zero
+  requests:
+    metric: requests
```

Comment:
Shouldn't it be Requests instead of requests ?

Reply:
Fixed